regression/regression_var.sh

```diff
@@ -4,6 +4,10 @@
 #  To run with hybrid ensemble option on, change HYBENS_GLOBAL and/or HYBENS_REGIONAL from "false" to "true".
 #  These are located at the end of this script.
 
+export local_or_default="/work/noaa/hwrf/noscrub/jcheng/gsi"
```

Comment:
Quick note:  Please remove lines 7 to 10.   I understand these are for your testing.  We don't want them committed to the authoritative repo.